assets/textures/gui/bevel/iron.png.import

```diff
@@ -3,23 +3,24 @@
 importer="texture"
 type="CompressedTexture2D"
 uid="uid://dlucsppj73dkc"
-path="res://.godot/imported/iron.png-7318fed7b3258c2bab4dfe321eb9db50.ctex"
+path.s3tc="res://.godot/imported/iron.png-7318fed7b3258c2bab4dfe321eb9db50.s3tc.ctex"
 metadata={
-"vram_texture": false
+"imported_formats": ["s3tc_bptc"],
+"vram_texture": true
```

Comment:
Hang on, is this a mistake? This is in the bevel folder telling me that this is a 2D GUI icon, meaning the changes here are invalid.

Reply:
Yes, that's a mistake. I'm not sure what happened there.

Reply:
:+1: 
You should revert the changes to this file.